components/admin_console/workspace-optimization/chips_list.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {FormattedMessage} from 'react-intl';
+
+import Chip from 'components/common/chip/chip';
+
+import './dashboard.scss';
+
+type ChipsInfoKeys = 'info' | 'warning' | 'error';
+
+export type ChipsInfoType = { [key in ChipsInfoKeys]: number };
```

Comment:
when we only use it like this we can also remove the extra type definition:

```suggestion
export type ChipsInfoType = { [key in 'info' | 'warning' | 'error']: number };
```

Reply:
ok